src/main.js

```diff
@@ -110,7 +113,7 @@ function storeInputsInArrays(coverInput, titleInput, descriptor1Input, descripto
 }
 
 function createNewCover(coverInput, titleInput, descriptor1Input, descriptor2Input) {
-  var newCover = new Cover(coverInput, titleInput, descriptor1Input, descriptor2Input)
+  currentCover = new Cover(coverInput, titleInput, descriptor1Input, descriptor2Input)
```

Comment:
not totally sure if this is correct because I have been struggling with this recently but these parameters could perhaps loose the 'Input' part of the names so that if/when a project starts to get big you prevent seeing the same word too much, which can become disorienting. I am trying to still find that balance of what verbs to use in naming that are not reserved words in Javascript

Reply:
Thanks so much for your review, Max! We really appreciate it!!!

Reply:
Thanks so much! Good call out.